common/src/main/java/com/wynntils/utils/mc/McUtils.java

```diff
@@ -31,6 +31,10 @@ public static LocalPlayer player() {
         return mc().player;
     }
 
+    public static String playerName() {
+        return player().getName().getString();
```

Comment:
I know it worked before but should Component#getString be used? Maybe StyledText#getStringWithoutFormatting is more stable

Reply:
I don't know... This is a pure refactoring, just shuffling code around. Can we keep it doing that, then we can experiment with player name formatting at some other time?

Reply:
Yes